src/app/test/factories/test-logger.factory.ts

```diff
@@ -0,0 +1,12 @@
+import { Logger } from '../../core/abstractions/logger.service'
+import { anyString, anything, instance, mock, when } from '@typestrong/ts-mockito'
+
+export class TestLoggerFactory {
+  public createLogger(): Logger {
```

Comment:
This could/should propable just be static, as it only meant to be used during tests.

Reply:
No, we just utilize that we use dependency injection.

Reply:
And "it's used by tests" is not a good enough argument.